tutorials/ai-core-and-LP-provisioning-GA/ai-core-and-LP-provisioning-GA.md

```diff
@@ -0,0 +1,118 @@
+---
+parser: v2
+auto_validation: true
+time: 20
+tags: [ tutorial>license, tutorial>beginner, topic>artificial-intelligence, topic>provisioning, software-product>sap-ai-core ]
+primary_tag: software-product>sap-ai-core
+author_name: Dhrubajyoti Paul
+author_profile: https://github.com/dhrubpaul
+---
+
+# Use Boosters for Free Tier Use of SAP AI Core and SAP AI Launchpad
+
```

Comment:
There should be a tutorial description right here.  The format is: <!-- description --> Description does after this comment.